### PR TITLE
refactor: rename `textfield-placeholder` to `label`

### PR DESCRIPTION
**Pull request by splincode**

### CODE REVIEW

File: projects/cdk/schematics/ng-update/v4/steps/migrate-styles.ts [hunk 1]
```diff
@@ -10,6 +11,7 @@ export function migrateStyles(): void {
             let fullText = sourceFile.getFullText();
 
             fullText = fullText
+                .replaceAll('textfield-placeholder()', 'label()')
```
Comment (vladimirpotekhin) [inline projects/cdk/schematics/ng-update/v4/steps/migrate-styles.ts:14]: there is no `tui` prefix, unfortunately, seems like it is better to not migrate this

Comment (splincode) [reply projects/cdk/schematics/ng-update/v4/steps/migrate-styles.ts]: fixed

**Review by vladimirpotekhin — APPROVED** [review]

File: projects/core/styles/mixins/textfield.less [hunk 2]
```diff
@@ -166,8 +166,7 @@
     padding-inline-end: 0.25rem;
 }
 
-// TODO: Rename to label in 4.0
-.textfield-placeholder() {
+.label() {
```
Comment (nsbarsukov) [inline projects/core/styles/mixins/textfield.less:169]: `textfield-label` ? 🤔 

(all 4 mixins above still has `textfield-` prefix)

File: projects/core/styles/mixins/textfield.scss [hunk 3]
```diff
@@ -157,8 +157,7 @@ $line-height-l: 1.25rem;
     padding-inline-end: 0.25rem;
 }
 
-// TODO: Rename to label in 4.0
-@mixin textfield-placeholder() {
+@mixin label() {
```
Comment (nsbarsukov) [inline projects/core/styles/mixins/textfield.scss:160]: Same question

Comment (splincode) [reply projects/core/styles/mixins/textfield.scss]: fixed

**Review by nsbarsukov — APPROVED** [review]

**Pull request merged**